frontend/src/hooks/queryBuilder/useGetExplorerQueryRange.ts

```diff
@@ -47,6 +48,7 @@ export const useGetExplorerQueryRange = (
 			selectedTime: 'GLOBAL_TIME',
 			globalSelectedInterval,
 			query: requestData || initialQueriesMap.metrics,
+			params,
```

Comment:
is options not working ?

Reply:
not really, I'm using it for fetch API params, not query hook params